### PR TITLE
Merge Freya.Integration into Freya.Pipeline #75

### PR DESCRIPTION
**Pull request by panesofglass**

### CODE REVIEW

File: src/Freya.Pipeline/Integration.fs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 //
 //----------------------------------------------------------------------------
 
-module Freya.Integration
+namespace Freya.Integration
```
Comment (kolektiv) [inline src/Freya.Pipeline/Integration.fs]: I think everywhere else is modules all the way down - is there a specific reason for the use of namespaces here?

**Comment by panesofglass** [comment]
No reason to make it a module as it only contains types and modules. Also, I was getting conflicts in the naming when using modules. For example, I couldn't name this `Freya.Pipeline.Integration` for some reason. Also, this harmonizes the OWIN related types and modules across multiple projects.

**Comment by kolektiv** [comment]
I'm looking at this now and thinking that it's slightly odd that some aspects of integration are in Core and some in Pipeline. It seems like the only reason is because the midfunc stuff needs to know about pipelines, and we've made pipeline a separate library, so we need to split the integration. Core, however, definitely does depend on OWIN, so it feels a bit off.

I wonder if we should in fact move the pipeline types (of which there are very few) and functions in to Core, and thus unify the integration in to Core too. Pipeline being outside core is a fairly arbitrary distinction, it's only a specific form of composition for the Freya<'a> type in essence, so there's nothing to stop us removing both Integration AND Pipeline as distinct projects and rolling the features of both in to Core.

What do you reckon @panesofglass ?

**Comment by panesofglass** [comment]
I'd be happy to roll them all into Core. I can't imagine that those using `Freya.Core` won't also use `Freya.Pipeline`, though I could be missing something. Let's merge them. Want me to do it?

**Comment by panesofglass** [comment]
I merged `Freya.Pipeline` into `Freya.Core` in PR #89

**Comment by kolektiv** [comment]
I'll take a look shortly :)

**Pull request merged**